app/Jobs/DispatchPollJobs.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Device;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
+use LibreNMS\Config;
+
+class DispatchPollJobs implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * @param  string  $device_spec
+     * @param  int  $verbosity
+     * @param  array<string, bool|string[]>  $module_overrides
+     */
+    public function __construct(
+        public string $device_spec,
```

Comment:
Let's assume this job is only used by the scheduled task. Other jobs would fill other needs. If code needs to be shared in the future it can be re-organized.

In that case, $device_spec, $verbosity, and $module_overrides are unused. We could probably remove them.

Reply:
I have immediate plans to use these, or do you think having duplicate functionality in the command is a better idea?

Reply:
I don't see it as duplicate functionality I think.  It feels like two very different use cases, with code that has to be flexible enough to handle both use cases. If you treat it as two different use cases, the code is a lot more streamlined. Perhaps that is what was bugging me about it.